src/styles/blocks/icon.scss

```diff
@@ -0,0 +1,61 @@
+/* #region icon */
+
+.icon {
+  display: block;
+  background-size: cover;
+
+  width: 24px;
+  height: 24px;
+
+  @include on-tablet {
+    width: 32px;
+    height: 32px;
+  }
+
+  @include on-desktop {
+    width: 32px;
+    height: 32px;
+  }
+
+  &:hover {
+    @include hover(transform, scale(1.2));
+  }
+
+  &--phone {
+    position: relative;
+    background-image: url("../images/icons/phone-call.svg");
+
+    &::before {
+      content: '+1 234 5555-55-5';
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 32px;
+      color: #000;
+      position: absolute;
+      width: 140px;
+      right: 50px;
+
+      transition: transform 0.5s ease;
```

Comment:
transition doesn't has the same speed as the rest of the others

![image](https://github.com/user-attachments/assets/e4732450-d349-4c7f-b5c1-06499486fd71)